ait/core/table.py

```diff
@@ -434,7 +433,7 @@ def create(self, name, *args):
         tab = None
         defn = self.get(name, None)
         if defn:
-            tab = createFSWTab(defn, *args)
+            tab = FSWTab(defn, *args)
```

Comment:
Sorry, my comment from last review was incorrect. The `createFSWxxx` method gets created in run-time during the extension process (i.e. `util.__init_extensions__`). So it is okay to call `createFSWxxx` method here. Please revert back to before this commit.